File: pinyougou/pinyougou-content/pinyougou-content-interface/src/main/java/com/pinyougou/content/service/ContentService.java
package com.pinyougou.content.service;

import com.pinyougou.pojo.TbContent;
import com.pinyougou.service.BaseService;
import com.pinyougou.vo.PageResult;

import java.util.List;

public interface ContentService extends BaseService<TbContent> {

    PageResult search(Integer page, Integer rows, TbContent content);

    List<TbContent> findContentListByCategoryId(Long categoryId);
}

File: pinyougou/pinyougou-portal-web/src/main/webapp/js/service/contentService.js
app.service("contentService", function ($http) {
    //根据分类 id 查询分类对应的内容列表
    this.findContentListByCategoryId = function (categoryId) {
        return $http.get("content/findContentListByCategoryId.do?categoryId="+categoryId);
    };
});